pyvistaqt/plotting.py

```diff
@@ -474,6 +474,7 @@ def close(self) -> None:
         if hasattr(self, "render_timer"):
             self.render_timer.stop()
         BasePlotter.close(self)
+        BasePlotter.deep_clean(self)
```

Comment:
Following the discussion on https://github.com/mne-tools/mne-python/pull/8406#issuecomment-715236419

I would not use `deep_clean` here until https://github.com/pyvista/pyvista/pull/959 is clarified

Reply:
Looks like it wasn't actually necessary anyway! Removed